.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!-- Write BELOW The Headers and ABOVE The comments else it may not be viewable. -->
+<!-- You can view CONTRIBUTING.md for a detailed description of the pull request process. -->
+
+## About The Pull Request
+
+<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage senior developers from merging your PR! -->
+
+## Why This Is Good For ClanGen
+
+<!-- If this is a bugfix, you can remove this section. -->
+<!-- Please add a short description of why you think these changes would benefit the game. -->
+<!-- If this PR was the result of discussion/testing on the discord, please add a link to the discord conversation here. -->
+<!-- If you have multiple features that can stand on their own, or unrelated bugfixes, please create separate PRs for them. -->
+
+## Linked Issues
+
+<!-- If this is not a bugfix, you can remove this section. -->
+<!-- If this was in response to a github issue, please write it here with the format Fixes: #1234 so that github knows to link the issues. -->
+<!-- If this PR was the result of discussion/testing on the discord, please add a link to the discord conversation here. -->
+
+## Proof of Testing
+
+<!-- Include any screenshots or debugging steps here. At least one form of proof of testing is REQUIRED for all new content. You must be able to run the code locally before you PR it here. -->
```

Comment:
I would add something about linking testing threads here as well, if one exists.  Perhaps change the first sentence to "Include any screenshots, debugging steps, or links to beta testing threads here."